include/drjit-core/jit.h

```diff
@@ -495,13 +495,17 @@ extern JIT_EXPORT uint32_t jit_registry_id(const void *ptr);
 
 /// Return the largest instance ID for the given domain
 /// If the \c domain is \c nullptr, it returns the number of active entries in
-/// all domains for the given variant.
+/// all domains for the given variant. If the \c variant is empty as well, it
+/// traverses all registry entries.
 extern JIT_EXPORT uint32_t jit_registry_id_bound(const char *variant,
                                                  const char *domain);
 
 /// Fills the \c dest pointer array with all pointers registered in the registry.
 /// \c dest must point to an array with \c jit_registry_id_bound(variant, nullptr) entries.
-extern JIT_EXPORT void jit_registry_get_pointers(const char *variant, void **dest);
+/// If variant is \c nullptr, it traverses all registry entries.
+extern JIT_EXPORT void jit_registry_get_pointers(const char *variant,
+                                                 const char *domain_name,
```

Comment:
```suggestion
                                                 const char *domain,
```

For consistency with the other functions.